data/856/326/21/85632621.geojson

```diff
@@ -404,8 +431,8 @@
         }
     ],
     "wof:id":85632621,
-    "wof:lastmodified":1587428801,
-    "wof:name":"Pinnacle Island",
+    "wof:lastmodified":1616015341,
+    "wof:name":"Senkaku Islands",
```

Comment:
Should the northern island shape be adjusted? 

![image](https://user-images.githubusercontent.com/853051/111574617-109deb00-876a-11eb-8636-64fe8c239fad.png)

Reply:
I redigitized this island, as well as a few others that are part of the Senaku Island area, we're good here.